lighthouse-core/lib/manifest-parser.js

```diff
@@ -460,10 +466,21 @@ function parse(string, manifestUrl, documentUrl) {
   };
   /* eslint-enable camelcase */
 
+  /** @type {string|undefined} */
+  let manifestUrlWarning;
+  try {
+    const manifestUrlParsed = new URL(manifestUrl);
+    if (!manifestUrlParsed.protocol.startsWith('http')) {
+      manifestUrlWarning = `WARNING: manifest URL not available over a valid network protocol`;
+    }
+  } catch (_) {
+    manifestUrlWarning = `ERROR: invalid manifest URL ${manifestUrl}`;
```

Comment:
if the url was completely invalid, it seems like this should be throwing, a la the undefined check at the top?

(in theory it really should never happen since *something* was fetched in order to get here)

Reply:
hm, yeah I agree in principle and it *should* never happen but seeing as it tanks the entire LH run, that seems like a bad outcome and it was reasonable to warn when the rest of the manifest can still be parsed?

Reply:
> hm, yeah I agree in principle and it should never happen but seeing as it tanks the entire LH run, that seems like a bad outcome and it was reasonable to warn when the rest of the manifest can still be parsed?

arg, very good point. I forgot it's not in a gatherer anymore.

I can't find anywhere that we surface the `warning` if `value` is defined, but we can think about that with the other manifest error stuff.